Code/Common/include/sitkImage.h

```diff
@@ -188,6 +188,7 @@ namespace simple
     std::vector< double > TransformContinuousIndexToPhysicalPoint( const std::vector< double > &index) const;
 
     std::vector< unsigned int > GetSize( void ) const;
+    unsigned int GetNumberOfPixels( void ) const;
```

Comment:
Please add documentation to clarify how this interacts with the number of components and for vector images.